src/indent.cpp

```diff
@@ -1933,6 +1933,16 @@ void indent_text(void)
          next = chunk_get_next(pc);
          if (next != nullptr)
          {
+            /*
+             * fixes  1260 , 1268 , 1277 (Extra indentation after line with multiple assignments)
+             * For multiple consecutive assignments in single line , the indent of all these
+             * assignments should be same and one more than this line's indent.
+             * so poping the previous assign and pushing the new one
+             */
+            if ((frm.pse[frm.pse_tos].type == CT_ASSIGN || frm.pse[frm.pse_tos].type == CT_ASSIGN_NL) && pc->type == CT_ASSIGN)
+            {
+               indent_pse_pop(frm, pc);
```

Comment:
Isn't it the same as avoiding doing the `indent_pse_push` from below?

Reply:
Yes Mihai, 
It will work if we avoid doing the indent_pse_push as well but,
the chunk that is responsible for indentation, which must be the last assignment operator in the series of consecutive assignments, should be in the stack.
Hence we are popping the unnecessary chunks and pushing the right one.